processor/src/operations/io_ops.rs

```diff
@@ -478,21 +478,21 @@ mod tests {
     fn op_pipe() {
         let mut process = Process::new_dummy_with_decoder_helpers_and_empty_stack();
 
-        // write words to the advice tape
+        // write words to the advice stack
```

Comment:
nit: let's change this to: "push words onto the advice stack".

Reply:
Done!